src/coinbase/types.ts

```diff
@@ -708,6 +764,14 @@ export enum StakeOptionsMode {
   NATIVE = "native",
 }
 
+export enum StakingOperationStatus {
```

Comment:
do we need this or is `StakingOperationStatusEnum` is good enough ?

Reply:
Yeah thought about it and does seem like there's any benefit to having an internal status representation. Removed it and put back the old `getStatus` function.